python/xnvme/tests/test_cython.pyx

```diff
@@ -0,0 +1,79 @@
+from cython.operator import dereference
```

Comment:
Optimally, we build these at the CI. But then we are already now forced to do binary wheels. I propose we let it stay like this, and solve it with the Cython wrapper